File: rest/conference/list-get-example-2/list-get-example-2.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'rubygems'          # This line not needed for ruby > 1.8
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = '{{ auth_token }}'
@client = Twilio::REST::Client.new account_sid, auth_token


# Loop over conferences and print out a property for each one
@client.account.conferences.list({
    :status => "in-progress",
    :friendly_name => "MyRoom"}).each do |conference|
    puts conference.status
end

File: rest/applications/list-get-example-2/list-get-example-2.js
// Download the Node helper library from twilio.com/docs/node/install
// These vars are your accountSid and authToken from twilio.com/user/account
var accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
var authToken = "{{ auth_token }}";
var client = require('twilio')(accountSid, authToken);

client.applications.list({ friendlyName: "MyApp" }, function(err, data) {
    data.apps.forEach(function(app) {
        console.log(app.VoiceUrl);
    });
});